src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +11,14 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String m = text.replaceAll("[a-z+A-Z+^!]", " ");
+        String[] tempString = m.trim().split("\\s+");
```

Comment:
try to do that inline without creating variable m(and let's give more informative name to our variable)

Reply:
done